scicomp/practical-git-prs.rst

```diff
@@ -115,6 +115,28 @@ How do you make a pull request
 .. _gh-pr: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/proposing-changes-to-your-work-with-pull-requests
 .. _gl-mr: https://docs.gitlab.com/ee/README.html#merge-requests
 
+Pull request description
+~~~~~~~~~~~~~~~~~~~~~~~~
+- These days, I (rkdarst) tend to write my initial PR message into my
+  commit, then ``git-pr`` will use that when I push.  This also stores
+  it permanent.
+- There is also the concept of "pull request templates" within
+  Github/Gitlab.  (After fast PRs via ``git-pr`` I really don't like
+  this being required)
```

Comment:
I would argue that pull request templates can be beneficial for projects which have a large number of contributors, either within the team or externally. A good example of that would be the A+ LMS: https://github.com/apluslms/.github/blob/master/pull_request_template.md

Then again, that might be slightly out of scope for this document.

Reply:
This is a good point, I was too opinionated there.  The templates do have a good purpose, especially for large feature changes.  My hesitation is biased because I have been using git-pr for making very many small changes, which (for example) that A+ template is too much.  (say, fix a typo, fix docs).  I think that these can be done quickly writing a free-form description (including the important things from the template).

Reply:
I think after the changes this looks very good, now the benefits of both tactics get mentioned